### PR TITLE
[Added] Danger to continuous integration

### PR DESCRIPTION
**Pull request by SD10**
If I set this up properly we should see a message from @SwifterSwiftBot. I accidently left an ugly commit so we'll have to squash that

### CODE REVIEW

File: Dangerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+message(“Thank you for submitting a pull request to SwifterSwift. The team will review your submission as soon as possible.“)
```
Comment (sunshinejr) [inline Dangerfile]: YOU'RE BETRAYING DANGER-SWIFT WITH THIS PEACE OF RUBY? 😭

Comment (SD10) [reply Dangerfile]: 🤣 You can see where my heart was in the previous commit. I can't even write Ruby. 
danger-swift is still blocked by Marathon but it's close!

**Comment by SD10** [comment]
cc @omaralbeik feel free to merge this if you're good with it!

File: Dangerfile (hunk 1 above)
Comment (orta) [reply Dangerfile]: Hah, yeah, if anyone wants to try help on that it's a real fun one  -https://github.com/JohnSundell/Marathon/pull/127

**Pull request merged**